README.md

```diff
@@ -1,2 +1,23 @@
+
+If labs do make the decision to adopt the NWB standard, they often find themselves motivated to convert their backlogged data to the NWB format. Recognizing the advantages of NWB, labs aim to ensure that their extensive historical datasets align with this standardized framework, enhancing data interoperability, collaboration, and long-term accessibility. However, this effort tends to be a substantial bottleneck in the transition to NWB. The accumulation of historical data, often stored in diverse and proprietary formats, poses a formidable challenge, as researchers must now navigate through and transform these datasets to comply with NWB. The sheer volume of legacy data, combined with its inherent heterogeneity, turns the conversion process into a labor-intensive endeavor. Further hampering adoption efforts is the dearth of individual incentives for researchers to prioritize this time-consuming task. In the fast-paced environment of scientific research, where immediate outcomes and results often take precedence, dedicating time and resources to the retrospective conversion of historical data may not be perceived as an urgent priority for individual researchers. The absence of tangible, career-advancing rewards or incentives further hampers the motivation to address this critical aspect of NWB adoption.
+
+Just as labs frequently seek to convert backlogged data, they are also often motivated to establish conversion pipelines to automatically convert newly generated data into NWB as they are recorded from acquisition systems. Automatically converting diverse neurophysiology data formats presents challenges due to the wide variety of modalities involved. Even a single lab may employ voltage recording, optical imaging, optogenetics, behavior, and more, each with its own software-dependent formats lacking standardization. A lab-specific conversion pipeline must take all the possible options into account to properly future-proof, or adapt itself to every new experiment that adds or removes a piece of software or hardware. These considerations make conversion pipelines using _pyNWB_ or _MatNWB_ alone difficult to develop and even harder to maintain for experimental researchers with many more immediate concerns vying for their attention.
```

Comment:
I would move this section outside of the intro - it's unsolved even as of 2024, so we don't want to make it seem like it will be answered by reading further - perhaps in a section towards the end talking about future developments?

Reply:
The reason I included it was to emphasize that neuroconv provides a programmatic scaffolding to _assist_ in the development of automatic conversion pipelines.  I don't think it needs to be completely solved. It just needs to be easier with the suite than it would be with pyNWB alone.

Reply:
Maybe the line selection from github doesn't show it right; I'm commenting on the second paragraph here

It's really not common yet for NWB conversion to occur "as they are recorded from acquisition systems"; in fact, usage of PyNWB (or realistically, `h5py` directly) is probably more useful in these cases than NeuroConv is

So I'd move it to a later discussion section as an idea/proposal of future work

Reply:
OK, I think the confusion here stems from the ambiguity of this paragraph. I was trying to refer to lab specific conversion pipelines that are developed in advance of data collection for some stereotyped experiments, which I believe are reasonably standard usage (see [A perspective on neuroscience data standardization with NWB: Pain points in the conversion workflow](https://arxiv.org/pdf/2310.04317.pdf)).  I will update the paragraph to try to clear that up.

Reply:
Yes, this reads better now. The point was not make it seem that the acquisition setup could directly output to NWB from the device using any of these packages